src/model/LottoMachine.js

```diff
@@ -0,0 +1,78 @@
+import {
+  getRandomNumber,
+  isDuplicated,
+  throwErrorWithCondition,
+  validate,
+} from "../utils/index.js";
+import Lotto from "./Lotto.js";
+
+class LottoMachine {
+  static #PRICE_PER_ONE = 1000;
+  #price;
+  #lottos;
+
+  constructor(price) {
+    LottoMachine.#validatePrice(price);
+    this.#price = price;
+    const count = LottoMachine.#countLotto(this.#price);
+    this.#lottos = LottoMachine.#createLottos(count);
+  }
+
+  get price() {
+    return this.#price;
+  }
+
+  get count() {
+    return this.#lottos.length;
+  }
+
+  get lottos() {
+    return [...this.#lottos];
+  }
+
+  static #createLottos(count) {
+    return Array.from({ length: count }).map(() => LottoMachine.#createLotto());
+  }
```

Comment:
```suggestion
  static #createLottos(count) {
    return Array.from({ length: count }).map(LottoMachine.#createLotto);
  }
```
아마 이렇게 표현해도 무방해보이네요!